### PR TITLE
Create shared as a workspace package to share stuff between server and client

### PR DESCRIPTION
**Pull request by dbertella**
Boom 💣 💥

### CODE REVIEW

File: packages/client/src/App.tsx [hunk 1]
```diff
@@ -2,6 +2,10 @@ import { Lobby } from 'pages/Lobby';
 import { Game } from 'pages/Game';
 import { Route, Switch } from 'react-router-dom';
 
+import { USER_EVENTS } from 'shared';
+
+console.log(USER_EVENTS);
```
Comment (stathismor) [inline packages/client/src/App.tsx]: Debug statement.

File: package.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "client": "yarn workspace client start",
     "server": "yarn workspace server start",
-    "start": "concurrently --kill-others-on-fail \"yarn server\"  \"yarn client\""
+    "shared": "yarn workspace server start",
```
Comment (stathismor) [inline package.json]: Why is this the same as `yarn server`? Does `shared` need to be part of any scripts anyway?

File: packages/shared/nodemon.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (stathismor) [inline packages/shared/nodemon.json]: Should nodemon be used in server? It's only for server at the moment.

File: packages/client/package.json [hunk 4]
```diff
@@ -20,7 +20,8 @@
     "socket.io-client": "^3.0.3",
     "theme-ui": "^0.3.4",
     "typescript": "^4.1.2",
-    "web-vitals": "^0.2.4"
+    "web-vitals": "^0.2.4",
```
Comment (stathismor) [inline packages/client/package.json:23]: What is this for?

**Review by stathismor — APPROVED** [review]

File: packages/shared/nodemon.json (hunk 3 above)
Comment (dbertella) [reply packages/shared/nodemon.json]: right no I need to remove this file

File: packages/client/package.json (hunk 4 above)
Comment (dbertella) [reply packages/client/package.json:23]: no idea some built in feature in CRA, it's something about analytics let's leave it there

**Pull request merged**